samples/rest/BingWebSearchV7.py

```diff
@@ -7,33 +7,41 @@
 import os
 from pprint import pprint
 import requests
+from dotenv import load_dotenv
+
+# Load the environment variables from .env file
+load_dotenv()
 
 '''
 This sample makes a call to the Bing Web Search API with a query and returns relevant web search.
 Documentation: https://docs.microsoft.com/en-us/bing/search-apis/bing-web-search/overview
 '''
+AUTH_HEADER=os.environ.get('BING_SEARCH_V7_AUTHORIZATION_HEADER', 'Ocp-Apim-Subscription-Key')
+API_KEY='BING_SEARCH_V7_WEB_SEARCH_SUBSCRIPTION_KEY'
 
 # Add your Bing Search V7 subscription key and endpoint to your environment variables.
```

Comment:
I suspect that the ability to set the endpoint URL via environment here is a holdover from Cognitive Services, where it was possible to customize the endpoint URL. That was not carried over into the current offering, so we should consider removing this option in the sample.

If we do so we should update the comments.

Reply:
Done: aaee62ec4862618af122513ffc88f05ec2669a79